tests/impl.cpp

```diff
@@ -3117,7 +3117,27 @@ result_t test_mm_blendv_epi8(const SSE2NEONTestImpl &impl, uint32_t i)
 
 result_t test_mm_slli_epi32(const SSE2NEONTestImpl &impl, uint32_t i)
 {
-    return TEST_UNIMPL;
+    const int32_t *_a = (const int32_t *) impl.mTestIntPointer1;
+    const int32_t *_b = (const int32_t *) impl.mTestIntPointer2;
+#if defined(__clang__)
+    // Clang compiler doesn't allow `_mm_slli_epi32()` take the second arg,
+    // count, greater than 31.
```

Comment:
Change to `Clang compiler does not allow the second argument of _mm_slli_epi32() to be greater than 31.`